prettier.config.cjs

```diff
@@ -0,0 +1,28 @@
+/** @typedef  {import("prettier").Config} PrettierConfig */
+
+/** @type { PrettierConfig | SortImportsConfig } */
+const config = {
```

Comment:
Thoughts on adding `importOrderSortSpecifiers: true` and `importOrderGroupNamespaceSpecifiers: true`?

Might also want `importOrderTypeScriptVersion`

Reply:
Funny enough I had some of these but prettier complained about not finding these optiosn so i removed it...

this is the config i usually roll: https://github.com/t3-oss/create-t3-turbo/blob/main/prettier.config.cjs

Reply:
Oh, It looks like 4.0.0 must have removed stuff then. They definitely still exist in 3.7.2

Reply:
yup: 
![image](https://github.com/pingdotgg/uploadthing/assets/11494384/aeea066c-8161-43d3-8d58-ccaf875fcb76)

Reply:
I'll enable separate type imports by eslint instead: 

Uploading CleanShot 2023-05-27 at 11.39.21.mp4…